homeassistant/components/homekit/__init__.py

```diff
@@ -866,11 +866,6 @@ async def async_stop(self, *args):
         self.status = STATUS_STOPPED
         _LOGGER.debug("Driver stop for %s", self._name)
         await self.driver.async_stop()
```

Comment:
https://github.com/ikalchev/HAP-python/blob/dev/pyhap/accessory_driver.py#L440
https://github.com/ikalchev/HAP-python/blob/dev/pyhap/accessory.py#L391

Since we are using the built-in stop functionality now we don't need to call it ourselves below